### PR TITLE
Add support to show multiple supervisor processes

### PR DESCRIPTION
**Pull request by rajasaur**
Show all the processes that failed as part of supervisor instead of stopping at the first one

### CODE REVIEW

File: bin/check-supervisor.rb [hunk 1]
```diff
@@ -64,10 +65,13 @@ def run
       critical "Tried to access #{config[:host]} but failed"
     end
 
+    failed_processes = Set.new
     @super.processes.each do |process|
-      critical "#{process['name']} not running: #{process['statename'].downcase}" if config[:critical].include?(process['statename'])
+      failed_processes << "#{process['name']} not running: #{process['statename'].downcase}" if config[:critical].include?(process['statename'])
     end
 
+    critical "\n#{failed_processes.to_a.join("\n")}" if not failed_processes.empty?
```
Comment (eheydrick) [inline bin/check-supervisor.rb]: use `unless` instead of `if not` here.

**Comment by eheydrick** [comment]
Can you add an entry to the CHANGELOG under unreleased and we'll get this merged.

**Comment by rajasaur** [comment]
@eheydrick : Thanks, added an entry in CHANGELOG and fixed the review comments

**Comment by eheydrick** [comment]
Thanks @rajasaur, can you also rebase against master so it can be merged.

**Comment by envintus** [comment]
@eheydrick is this still eligible to be merged? Looks like @rajasaur rebased, Travis CI build passed, and I see no additional feedback or requests made to him.

**Comment by envintus** [comment]
@sstarcher Thanks!!

**Pull request merged**